src/kirby/components/list/list.component.scss

```diff
@@ -5,19 +5,37 @@ ul {
   margin-top: 0;
   font-size: font-size('s');
   margin-block-end: 0;
+
   li {
-    background-color: white;
     background-color: var(--kirby-surface);
     border-bottom: 1px solid var(--kirby-background);
     padding: 16px;
```

Comment:
```suggestion
    padding:  size('m');
```

Reply:
I think this was left out when Peter merged my branch onto his 😄

Reply:
The hover color could use some transparancy, hence the thing I tried with rgba 😄

Reply:
Should that not be a themed variable if it should have transparancy? or a lighter version of `--kirby-secondary` ?